src/Command/CodeStudio/CodeStudioWizardCommand.php

```diff
@@ -68,7 +68,24 @@ protected function execute(InputInterface $input, OutputInterface $output) {
     $acquia_cloud_client = $this->cloudApiClientService->getClient();
     $account_adapter = new Account($acquia_cloud_client);
     $account = $account_adapter->get();
-    $this->validateRequiredCloudPermissions($acquia_cloud_client, $this->appUuid, $account);
+    $this->validateRequiredCloudPermissions(
+      $acquia_cloud_client,
+      $this->appUuid,
+      $account,
+      [
+        "deploy to non-prod",
+        # Add SSH key to git repository
+        "add ssh key to git",
+        # Add SSH key to non-production environments
+        "add ssh key to non-prod",
+        # Add a CD environment
```

Comment:
CD? No, IDE you mean, right?

Reply:
This line is for the codestudio:wizard command, so CD is correct.